nvflare/dashboard/application/store.py

```diff
@@ -254,10 +251,9 @@ def patch_client_by_creator(cls, id, req):
             client.organization_id = org.id
 
         capacity = req.pop("capacity", None)
-        if capacity is not None:
-            capacity = json.dumps(capacity)
-            cap = get_or_create(db.session, Capacity, capacity=capacity)
-            client.capacity_id = cap.id
+        if capacity:
+            print(f"from creator: {capacity=}")
```

Comment:
Do we need this print or this is for debugging purpose

Reply:
Oh, this is just for debug. Will remove.